backend/chromeApiServer/src/main/java/com/canvas/controllers/chromeApiServer/ChromeApiController.java

```diff
@@ -47,11 +53,25 @@ public ResponseEntity<CommandOutput> initiateInstructorCodeEvaluation(
             @PathVariable("courseId") String courseId,
             @PathVariable("studentId") String studentId,
             @RequestParam("userType") UserType type
-    ) throws IOException, UserNotAuthorizedException {
+    ) throws UserNotAuthorizedException, IncorrectRequestParamsException, CanvasAPIException {
+        // Check request params are correct, not null or empty.
+        // TODO: What do we consider incorrect? Define further
+        if (bearerToken == null || assignmentId == null || courseId == null || studentId == null || type == null) {
```

Comment:
In a future CR we can create a function like "isValid()" that does all these null checks to clean the code up